packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -1244,6 +1259,139 @@ public void setBackfaceVisibilityDependantOpacity() {
     setAlpha(0);
   }
 
+  public void initializeFocusGuideTalkbackAccessibilityDelegate() {
+    if (!this.isTVFocusGuide()) {
+      return;
+    }
+
+    this.originalIsFocusable = this.isFocusable();
+
+    ReactAccessibilityDelegate viewAccessibilityDelegate = new ReactAccessibilityDelegate(
+      this, originalIsFocusable, this.getImportantForAccessibility()
+    ) {
+      @Override
+      public boolean performAccessibilityAction(View host, int action, Bundle args) {
+        if (!(host instanceof ReactViewGroup self)) {
+          return super.performAccessibilityAction(host, action, args);
+        }
+        if (action == AccessibilityNodeInfo.ACTION_FOCUS) {
+          if (self.interceptAccessibilityEvents(action, args)) {
+            return true;
+          }
+          // Handle case when focus guide cannot find any focusable child
+          if (self.isTVFocusGuide() && self.getFirstFocusableView(self) == null) {
```

Comment:
Verified that the changes to handle no focusable children caused the regression I'm seeing.

Reply:
I'll try to investigate today's evening, but I'll need some more time, so maybe it's not worth to include it in 0.76.0

Reply:
That's fine, thanks for taking a look!